typescript/cli/src/commands/deploy.ts

```diff
@@ -110,6 +112,49 @@ const coreCommand: CommandModuleWithWriteContext<{
   },
 };
 
+/**
+ * Generates a command module for deploying Hyperlane contracts, given a command
+ *
+ * @param commandName - the deploy command key used to look up the deployFunction
+ * @returns A command module used to deploy Hyperlane contracts.
+ */
+export const deployCore: CommandModuleWithWriteContext<{
+  chain: string;
+  config: string;
+  dryRun: string;
+  fromAddress: string;
```

Comment:
is this required in dry run? seems unused

Reply:
yeah, its used in `contextMiddlware`